source/Windows.Devices.I2c/I2cController.cs

```diff
@@ -64,10 +90,7 @@ public static I2cController GetDefault()
             {
                 lock (_syncLock)
                 {
-                    if (s_instance == null)
-                    {
-                        s_instance = new I2cController();
-                    }
+                    s_instance = new I2cController();
```

Comment:
You need to keep the lazy init pattern here. After entering the lock you have to check again if the instance is still null.
In case "this" was the waiting thread, the object could already have been created elsewhere.